classes/collectors/GoodActsReactionCollector.js

```diff
@@ -123,6 +123,7 @@ export class GoodActsReactionCollector extends ReactionCollectorBase {
 			//award GP to msg author
 			else await this.awardApprovalGP({
 				user: msg.author,
+				approver: user.tag,
```

Comment:
```suggestion
				approver: user,
```
The pattern I'd like to follow is to pass the entire user object, then extract what's needed in the called function